example/basic/views/fn.js

```diff
@@ -1 +1 @@
-console.log('the fn.js file/code gets included in the "compiled" JS file with all the tags!');
+console.log('the fn.js file/code gets included in the "compiled" JS file with all the tags!'); // eslint-disable-line
```

Comment:
ok, this is "_justified_" as an example ... 
(_as much as it pains me to see: `// eslint-disable-line`, you're just fixing my example code... thanks!_)